app/src/components/market/sections/market_list/market_home_container.tsx

```diff
@@ -236,7 +236,7 @@ const MarketHomeContainer: React.FC = () => {
   const defaultCpkAddress = context.relay ? context.account : null
   const [cpkAddress, setCpkAddress] = useState<Maybe<string>>(defaultCpkAddress)
 
-  const [pageSize, setPageSize] = useState(4)
+  const pageSize = 12
```

Comment:
Super minor, but since this is a constant now it's best practice to use `CONSTANT_CASE`

Reply:
im learning everyday, thanks ^.^